security/acme-client/src/opnsense/mvc/app/controllers/OPNsense/AcmeClient/forms/dialogValidation.xml

```diff
@@ -712,6 +712,36 @@
         <type>textbox</type>
         <help>Requires the whole key file in a format that is compatible with nsupdate.</help>
     </field>
+    <field>
+        <label>opnsense</label>
+        <type>header</type>
+        <style>table_dns table_dns_opnsense</style>
+    </field>
+    <field>
+        <label><![CDATA[Please manually install and configure the plugin "bind".]]></label>
```

Comment:
```suggestion
        <label>Please manually install and configure the bind plugin.</label>
```

Simplify and remove quotes would make it imho better.

Reply:
OK with me